public/stylesheets/admin.css

```diff
@@ -0,0 +1,55 @@
+body {
+  font-family: 'Amatic SC', sans-serif;
+  margin-top: 25px;
+  margin-bottom: 25px;
+}
+
+.footer {
+  display: flex;
+  justify-content: space-between;
+  font-size: 1.5em;
+}
+
+.footer button {
+  width: 100px;
+  height: 40px;
+  border: 1px black solid;
+  color: white;
+  font-family: 'Amatic SC', sans-serif;
+  font-size: 24px;
+  background-color: black;
+}
+
+.navbar {
+  display: flex;
```

Comment:
I tried the display flex properties off the navbar and it looked and behaved the same.